plugins/Live/Reports/GetLastVisitsDetailsWidget.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Copyright (C) Piwik PRO - All rights reserved.
+ *
+ * Using this code requires that you first get a license from Piwik PRO.
+ * Unauthorized copying of this file, via any medium is strictly prohibited.
+ *
+ * @link http://piwik.pro
+ */
+
+namespace Piwik\Plugins\Live\Reports;
+
+use Piwik\Plugins\Live\Visualizations\VisitorLogWidget;
+
+class GetLastVisitsDetailsWidget extends Base
```

Comment:
OK I got it. It was changed to use Report / Visualization class etc so it can use the event I presume. It's not meant to be a new widget.